src/documentation/ToolkitDocumentation/model.ts

```diff
@@ -5,44 +5,55 @@
  */
 
 export interface Toolkit {
+  id: string;
   name: string;
   description: string;
-  contents: ToolkitTopic[];
+  contents?: ToolkitTopic[];
 }
 
 export interface ToolkitTopic {
   name: string;
   /**
    * Short, for the listing.
    */
-  description: string;
+  subtitle: string;
   /**
    * Longer, for the heading above the contents.
    */
   introduction?: string;
-  contents: ToolkitTopicItem[];
+  contents?: ToolkitTopicEntry[];
 }
 
 export interface ToolkitCode {
-  _type: "code";
-  detail?: boolean;
-  value: string;
-  select?: {
-    prompt: string;
-    options: string[];
-    placeholder: string;
-  };
+  _type: "python";
+  main: string;
 }
 
 export interface ToolkitText {
-  _type: "text";
-  detail?: boolean;
-  value: string;
+  _type: "block";
+}
+
+export interface ToolkitImage {
+  _type: "simpleImage";
+  // More!
+}
```

Comment:
In practice we're not using these types. Let's revisit the typing here and aim to be more consistent - it might be that a marker for portable text is all that's worth doing and we can keep it untyped.

Reply:
Or we use them in the serializers to type `node`.

Reply:
Rationalised in d5c801b85315bff8b6e8e563d29c3bd700da3723